.github/workflows/integration.yml

```diff
@@ -55,7 +39,7 @@ jobs:
           min_capi=($(yq .cf-d-capi-version-min build_data.yml))
           echo "Minimum CAPI version: ${min_capi}"
           echo "min-capi=${min_capi}" >> $GITHUB_OUTPUT
-
+        
```

Comment:
We should consider linting our workflows too.

Reply:
Will add story for this in tracker.